src/types/Account/methods/getTransaction.js

```diff
@@ -30,13 +32,26 @@ async function getTransaction(txid = null) {
     chainLocked,
   };
   if (this.cacheTx) {
+    // We cache even if transaction / metadata are not final (case of unconfirmed tx)
     await this.importTransactions([[transaction, metadata]]);
```

Comment:
Yes we do. 
We actually won't go trying to decide for the user what is important and what is not. 
If he requires a transaction, it must be for some reason (for instance, it may be parent tx of a tx for full inputs of outputs knowledge), and if he ask to cache it, then let's cache it. 
If he wants to remove it, he can :)

Reply:
So let’s assume user has an app like block explorer and this app uses SDK and wallet to get transaction info. So we gonna fetch all headers for all transactions that the user sees in block explorer and cache them?

Reply:
In this case, cacheTx will be set to false (it's an option)

Reply:
I see. Thank you.